toptal/picasso
# feat(Tooltip): add maxWidth prop

### Description

We have tooltips with decent amount of content so that the default `maxWidth=300px` is not enough.

### How to test

- https://picasso.toptal.net/add-max-width-property-to-tooltip/?path=/story/overlays-folder--tooltip
- scroll to maxWidth section and ensure it works good


### Screenshots

<img width="700" alt="Screen Shot 2020-09-04 at 17 02 48" src="https://user-images.githubusercontent.com/14301985/92247876-84645e80-eed0-11ea-8d50-257c8914a186.png">

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation


</details>
